lib/protoboeuf/google/protobuf/any.rb

```diff
@@ -553,10 +553,26 @@ def _encode(buff)
 
         def to_h
           result = {}
-          result["type_url".to_sym] = @type_url
-          result["value".to_sym] = @value
+
+          result[:"type_url"] = @type_url
+          result[:"value"] = @value
```

Comment:
I don't know why that `to_sym` call was there to begin with, this looks a little funny to me. `result[:type_url]` and `result[:value]` would be much more natural.

Reply:
I left the quotes just in case the field names have anything funky in them that might produce a syntax error.